index.js

```diff
@@ -17,12 +17,13 @@ module.exports = {
       parent = parent.app;
     }
 
-    let fontAwesomeToImport = fs.readdirSync(`${this.nodeModulesPath}/font-awesome/fonts`);
+    const nodeModulesPath = parent.project.nodeModulesPath;
```

Comment:
For backwards ( WAY back ) compatibility, should we `const nodeModulesPath = parent.project.nodeModulesPath || this.nodeModulesPath;` ?